en/docs/guides/user-mgt/tenant-wise-config.md

```diff
@@ -0,0 +1,132 @@
+# Tenant-Wise Email Sender Configuration
+
+When handling notifications such as the ones given below, the email-sender configuration needs to be changed in `<IS-HOME>/repository/conf/deployment.toml`. 
+
+- [EmailOTP](insert-link-admin-portal)
+- [Password Recovery](insert-link-admin-portal)
+- [Username Recovery](insert-link-admin-portal)
+- [Creating Users using the Ask PasswordOption](../../guides/user-mgt/ask-password)
+
+
+**Email configurations**
+
+ ``` toml
+ [output_adapter.email]
+ from_address= "wso2iamtest@gmail.com"
+ username= "wso2iamtest"
+ password= "Wso2@iam70"
+ hostname= smtp.gmail.com
+ port= 587
+ enable_start_tls= true
+ enable_authentication= true
+ ```
+ However, this configuration will apply to all the tenants. If you wish to configure them tenant-wise, follow the instructions given below. 
+ 
+1. Configure the [Configuration Management REST API](insert-rest-api-link). 
+2. Execute the following curl command to create a resource type named `Publisher`. 
+
+    **Sample Request**
+    ``` java 
+    curl -X POST "https://localhost:9443/t/{tenant-domain}/api/identity/config-mgt/v1.0/resource-type" -H "accept: 
+    application/json" -H 
+    "Content-Type: application/json" -H 'Authorization: Basic YWRtaW46YWRtaW4=' -d "{ \"name\": \"Publisher\", \"description\": \"Publisher Configurations\"}"
+    ```
+3. Execute the following curl command for creating a resource named `EmailPublisher`. 
+
+    **Sample Request**
+    ``` java 
+    curl -X POST "https://localhost:9443/t/{tenant-domain}/api/identity/config-mgt/v1.0/resource/Publisher" -H "accept: 
+    application/json" -H "Content-Type: application/json" -H 'Authorization: Basic YWRtaW46YWRtaW4=' -d "{ \"name\": \"EmailPublisher\", \"attributes\": [ { \"key\": \"email\", \"value\": \"string\" } ]}"
+    ```
+4. Execute the following curl command for creating a file named `EmailPublisher`. 
+
+    **Sample Request**
+    ``` java 
+    curl -X POST "https://localhost:9443/t/{tenant-domain}/api/identity/config-mgt/v1
+    .0/resource/Publisher/EmailPublisher/file" -H "accept: application/json" -H 
+    "Content-Type: multipart/form-data" -H 'Authorization: Basic YWRtaW46YWRtaW4=' -F "resourceFile=@EmailPublisher.xml;type=text/xml" -F "fileName=EmailPublisher"
+    ```
+    This `EmailPublisher.xml` file will be used as the tenant's email publisher file. We can configure the tenant-wise email configurations here. 
+    
```

Comment:
avoid using "we". 

we can say "configure the tenant-wise email configurations in the 'EmailPublisher.xml' file.